Hogajama/hogajama-rs/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,11 @@
+mp.messaging.connector.smallrye-kafka.bootstrap.servers=${kafka.bootstrap.servers}
+
+mp.messaging.incoming.habarama-in.connector=smallrye-kafka
+mp.messaging.incoming.habarama-in.topic=habarama
+mp.messaging.incoming.habarama-in.merge=true
+mp.messaging.incoming.habarama-in.broadcast=true
```

Comment:
The incoming stream can not be broadcasting, this statement will be ignored. Setting merge and broadcast in a single stream was introduced to the config when I defined the incoming and outgoing streams as a single connector-stream (which didn't work somehow...). I should have cleaned this up two weeks ago 👿

Reply:
In accordance to https://smallrye.io/smallrye-reactive-messaging/smallrye-reactive-messaging/2/model/model.html 

The Broadcast annotation changes this behavior and indicates that messages transiting in the channel are **dispatched to all the consumers**

The Merge annotation changes this behavior and indicates that a channel **can have multiple producers.**

The habarama consumer needs the enabled **merge property** if we want to be able to receive message from multiple amq topics. As of right now, we have a one-to-one connection, so we actually could remove both properties.